### DISCUSSION TITLE
How can I customize the tailwind-light theme with the visual editor?

### DISCUSSION THREAD

**Discussion by aaron-za-dev** [primefaces/community#226] [PrimeNG] [answered]
I want to use the tailwind theme as a base to create my own theme, but it is not available in the visual editor.

![image](https://github.com/primefaces/community/assets/16365508/15a7f42f-edec-45f7-920b-b5b3481bf1dd)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

The current designer app is outdated and will not receive an update. We're currently working on a new Prime Designer App to replace the old one.

Because of that, we've made theme designer open-source you can see it here https://github.com/primefaces/primeng-sass-theme

To customize the tailwind light theme follow these steps;

1-) Clone the repo I've shared above.
2-) Navigate to `themes/tailwind/tailwind-light/_variables.scss`, and you'll see all the theme variables.
3-) Modify the variables as you wish.
5-) run `sass --update themes:themes --no-source-map` command.
6-) Your theme file is ready in theme.css, copy and paste it into your project.
